custom_components/powercalc/data/bose/416776/model.json

```diff
@@ -0,0 +1,30 @@
+{
+    "calculation_enabled_condition": "{{ is_state('[[entity]]', 'playing') }}",
+    "calculation_strategy": "linear",
+    "device_type": "smart_speaker",
+    "linear_config": {
+        "calibrate": [
+            "10 -> 2.94",
+            "20 -> 2.94",
+            "30 -> 2.94",
+            "40 -> 2.94",
+            "50 -> 3.03",
+            "60 -> 3.2",
+            "70 -> 3.49",
+            "80 -> 4.57",
+            "90 -> 6.96",
+            "100 -> 8.54",
+            "0 -> 3.15"
```

Comment:
Can you remeasure this value manually? 
This is the value for when the speaker is muted, so you can click mute on the media player control in HA.
It should be lower or equal to 2.94 looking at the other measurements.
I think this might be caused by sleep time not being enough and the power meter setteling coming from the 100 volume.